compiler/rustc_middle/src/traits/mod.rs

```diff
@@ -923,7 +923,7 @@ impl ObjectSafetyViolation {
                 }
             }
             ObjectSafetyViolation::SupertraitNonLifetimeBinder(_) => {
-                format!("where clause cannot reference non-lifetime `for<...>` variables").into()
```

Comment:
who wrote this :forgor:

Reply:
git blame now points at me, so it's clearly my fault